cdm/core/src/main/java/ucar/nc2/dataset/CoordinateAxis1D.java

```diff
@@ -972,10 +972,18 @@ private boolean makeBoundsFromAux() {
   private void makeEdges() {
     int size = (int) getSize();
     edge = new double[size + 1];
-    if (size < 1)
+    if (size < 1) {
       return;
-    for (int i = 1; i < size; i++)
+    }
+    if (size == 1) {
+      // if the axis size is zero, edges are the same as the coord
+      edge[0] = coords[0];
+      edge[1] = coords[0];
+      return;
+    }
+    for (int i = 1; i < size; i++) {
       edge[i] = (coords[i - 1] + coords[i]) / 2;
+    }
     edge[0] = coords[0] - (edge[1] - coords[0]);
     edge[size] = coords[size - 1] + (coords[size - 1] - edge[size - 1]);
     isContiguous = true;
```

Comment:
should this be set to true in the case `size == 1`? The default is also true so not sure if it is needed

Reply:
I don't think it matters when there's just one point. Contiguous is meaningless then, and (I think) we're handling it further up the pipeline well enough that it works.